tests/lib/outputs/json_ocsf/json_ocsf_test.py

```diff
@@ -1,242 +0,0 @@
-        assert finding_json_ocsf.status_id == StatusID.Other.value
-        assert finding_json_ocsf.status == StatusID.Other.name
-        assert finding_json_ocsf.status_code == finding_output.status
-        assert finding_json_ocsf.status_detail == finding_output.status_extended
-
-    # Returns TypeID.AWS_Account when provider is 'aws'
-    def test_returns_aws_account_when_provider_is_aws(self):
-        provider = "aws"
-        result = get_account_type_id_by_provider(provider)
-
-        assert result == TypeID.AWS_Account
-
-    # Returns TypeID.Azure_AD_Account when provider is 'azure'
-    def test_returns_azure_ad_account_when_provider_is_azure(self):
-        provider = "azure"
-        result = get_account_type_id_by_provider(provider)
-
-        assert result == TypeID.Azure_AD_Account
-
-    # Returns TypeID.GCP_Account when provider is 'gcp'
-    def test_returns_gcp_account_when_provider_is_gcp(self):
-        provider = "gcp"
-        result = get_account_type_id_by_provider(provider)
-
-        assert result == TypeID.GCP_Account
-
-    # Returns TypeID.Other when provider is None
-    def test_returns_other_when_provider_is_none(self):
-        provider = None
-        result = get_account_type_id_by_provider(provider)
-
-        assert result == TypeID.Other
-
-    # Returns StatusID.New when status is "FAIL" and muted is False
-    def test_new_when_status_fail_and_not_muted(self):
-        status = "FAIL"
-        muted = False
-        result = get_finding_status_id(status, muted)
-
-        assert result == StatusID.New
-
-    # Returns StatusID.Suppressed when status is "FAIL" and muted is True
-    def test_suppressed_when_status_fail_and_muted(self):
-        status = "FAIL"
-        muted = True
-        result = get_finding_status_id(status, muted)
-
-        assert result == StatusID.Suppressed
-
-    # Returns StatusID.Other when status is None and muted is False
-    def test_other_when_status_whatever_and_not_muted(self):
-        status = None
-        muted = False
-        result = get_finding_status_id(status, muted)
-
-        assert result == StatusID.Other
-
-    # Returns StatusID.Suppresed when status is None and muted is True
-    def test_other_when_status_whatever_and_muted(self):
-        status = None
-        muted = True
-        result = get_finding_status_id(status, muted)
-
-        assert result == StatusID.Suppressed
-
-    # Returns StatusID.Suppressed when muted is True and status is not "FAIL"
-    def test_suppressed_when_status_pass_and_muted(self):
-        status = "PASS"
-        muted = True
-        result = get_finding_status_id(status, muted)
-
-        assert result == StatusID.Suppressed
-
-    # Returns StatusID.Other when muted is False and status is not "FAIL"
-    def test_other_when_status_pass_and_not_muted(self):
-        status = "PASS"
-        muted = False
-        result = get_finding_status_id(status, muted)
-
-        assert result == StatusID.Other
```

Comment:
Please move this tests to the new test class, you don't need to create a `ASFF` class just call the static methods with `ASFF.method(params)`

Reply:
Done